x/gov/client/rest/grpc_query_test.go

```diff
@@ -52,9 +49,11 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	_, err = govtestutil.MsgVote(val.ClientCtx, val.Address.String(), "1", "yes")
 	s.Require().NoError(err)
 
-	// create a proposal without deposit
+	// create a proposal with min initial deposit
+	minInitialDeposit := govtestutil.GetMinInitialValidDeposit(types.DefaultDepositParams().MinDeposit)
 	_, err = govtestutil.MsgSubmitProposal(val.ClientCtx, val.Address.String(),
-		"Text Proposal 2", "Where is the title!?", types.ProposalTypeText)
+		"Text Proposal 2", "Where is the title!?", types.ProposalTypeText,
```

Comment:
Can we change this from saying "Where is the title", the title is Text Proposal 2.

Reply:
On another look, I would prefer not to touch this because these tests are coupled together. I'm afraid that changing this will break a lot of other things that we don't have time to fix right   now